install.sh

```diff
@@ -313,7 +311,7 @@ install_trdl() {
   is_trdl_installed_and_up_to_date "$trdl_version" && log::info "Skipping trdl installation: already installed in \"$HOME/bin/\"." && return 0
   log::info "Installing trdl to \"$HOME/bin/\"."
 
-  run_as_user "$user" "mkdir -p \"$HOME/bin\"" || abort "Can't create \"$HOME/bin\" directory."
+  run_as_user "$user" "mkdir -p $HOME/bin" || abort "Can't create \"$HOME/bin\" directory."
```

Comment:
I advise to **always** enquote all strings, be it path, URI, ..., maybe except very simple static strings. Don't think about whether you need it or not, just always add quotes, gonna be simpler that way.

This command will break on `mkdir -p /home/Ilya Lesikov/bin` now.